bindings/wasm/src/account/types/encryption_algorithm.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright 2020-2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use identity::account_storage::EncryptionAlgorithm;
+use serde::Deserialize;
+use serde::Serialize;
+use wasm_bindgen::prelude::*;
+
+use crate::error::Result;
+use crate::error::WasmResult;
+
+/// Supported keys for encrypting data
+#[derive(Clone, Serialize, Deserialize)]
+#[wasm_bindgen(js_name = EncryptionAlgorithm, inspectable)]
+pub struct WasmEncryptionAlgorithm(EncryptionAlgorithm);
+
+#[wasm_bindgen(js_class = EncryptionAlgorithm)]
+impl WasmEncryptionAlgorithm {
+  /// Encrypts/Decrypts data using Aes256Gcm.
+  #[wasm_bindgen(js_name = aes256gcm)]
+  pub fn aes256gcm() -> WasmEncryptionAlgorithm {
```

Comment:
Q: do we intend to follow `JWA` naming schemes, should we use `Aes256Gcm` or `A256GCM` as per JWA? If we keep the former then we should probably use `EcdhEs`? Doesn't really matter but we should just be consistent.